doc/nrf/libraries/others/emds.rst

```diff
@@ -64,7 +64,7 @@ The :c:func:`emds_is_ready` function can be called to check if EMDS is prepared
 Once the data storage has completed, a callback is called if provided in :c:func:`emds_init`.
 This callback notifies the application that the data storage has completed, and can be used to reboot the CPU or execute another function that is needed.
 
-After completion of :c:func:`emds_store`, the :c:func:`emds_is_ready` function call will return error, since it can no longer guarantee that the data will fit into the flash area.
+After completion of :c:func:`emds_store`, the :c:func:`emds_is_ready` function call will return error, since it can no longer guarantee that the data will fit into the persistent memory area.
```

Comment:
```suggestion
After completion of :c:func:`emds_store`, the :c:func:`emds_is_ready` function call will return an error, because it can no longer guarantee that the data will fit into the persistent memory area.
```